evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md

```diff
@@ -0,0 +1,59 @@
+| **Report Date** | Date of submission (2024/09/25) |
+| **Submitted by**| Edward Mack                     |
+
+
+## Member details
+
+- Matrix username: @mackcom:matrix.org
+- Polkadot address: 1ZYq7qWxvqC1rE4PKnTWwoVJYLtn99p9JM9cUn2WqbWr3Jt
+- Current rank: 2
+- Date of initial induction:
+- Date of last report: NA
+- Area(s) of Expertise/Interest: Gossamer node, parachain subsystems 
+
+
+## Reporting period
+
+- Start date: 2024/07/01
+- End date: 2024/09/25
+
+
+## Evidence
+In the past three months, I have been working on the Candidate Validation subsystem for the Gossamer node. This 
+includes implementing code for handing validation from exhaustive, validation from chainstate, pvf host development 
+and work on validation precheck.  Details can be found in the following PRs:
+
+| Areas of Contribution                         | Tasks                                                               | Links                                                                |Notes |
+|-----------------------------------------------|---------------------------------------------------------------------|----------------------------------------------------------------------|--|
+| Gossamer node, candidate validation subsystem | Implement validate candidate from exhaustive                        | [ChainSafe PR 4036](https://github.com/ChainSafe/gossamer/pull/4036) |  |
+| Gossamer node, candidate validation subsystem | Implement candidate validation from chainstate                      | [ChainSafe PR 4067](https://github.com/ChainSafe/gossamer/pull/4067) |  |
+| Gossamer node, candidate validation subsystem | Create worker pool for PVF host                                     | [ChainSafe PR 4101](https://github.com/ChainSafe/gossamer/pull/4101) |  |
+| Gossamer node, candidate validation subsystem | Confirm invalidity errors are implemented in candidate validation   | [ChainSafe PR 4163](https://github.com/ChainSafe/gossamer/pull/4163) |  |
+
+
+## Voting record
+*Provide your voting record in relation to required thresholds for your rank.* 
+
+|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities                                                                                                                                                                                                                                                                                               | Comments |
+|---|---|---|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-|
+|III|70%   |100%  | I have voted on 0 out of 2 referenda in which I was eligible to vote (i.e 0 % voting activity). I intend to be more active regarding voting in the future.  | |
```

Comment:
Please do not override the example. Instead, you need to fill the row of the table that corresponds to your rank.
See an example of how this should be done below:
![image](https://github.com/user-attachments/assets/2e0d741c-504a-4833-a1e6-ef8b3bcf9ddd)

Reply:
I've updated this, thanks for providing an example.